backend/apid/graphql/globalid/event_filters.go

```diff
@@ -0,0 +1,23 @@
+package globalid
+
+import "github.com/sensu/sensu-go/types"
+
+//
+// Event Filters
+//
+
+var eventFilterName = "eventFilters"
```

Comment:
Eventually I think I want to do away with this package, however, until then I think it might be nice to have this match the resource name.. https://github.com/sensu/sensu-go/blob/master/api/core/v2/filter.go#L16